src/app/help/en/welcome.md

```diff
@@ -164,24 +163,24 @@ Once you follow someone, all of their posts will show up in your "Feed" on the h
 
 As other Steemians come across your posts and comments, you will start to gain followers.
 
-You can see all of your followers and the people you are following in your profile page.
+You can see all of your followers and the people you are following on your profile page.
 
 <a href="#Table_of_Contents">^</a>
 ### <span id="Resteem">Resteem</span>
 
-If you want to share someone else's post with all of your followers, click on the <Icon name="reblog" /> *resteem* icon.
+If you want to share someone else's post with all of your followers, click on the <Icon name="reblog" /> *resteem* icon. Resteems cannot be undone. 
 
 <a href="#Table_of_Contents">^</a>
 ### <span id="Digital_Currencies">Digital Currencies</span>
 
-STEEM, Steem Power and Steem Dollars are the three forms of digital currency used by the Steem Blockchain.
+STEEM, Steem Power and Steem Dollars are the three forms of digital currency tokens used by the Steem Blockchain.
 
 More information on the three types of tokens can be found in the [Steemit FAQ](https://steemit.com/faq.html#What_is_the_difference_between_STEEM__STEEM_Power__and_Steem_Dollars).
 
 <a href="#Table_of_Contents">^</a>
 ### <span id="Curation">Curation</span>
 
-Up to 25% of the reward for posts goes to the people who voted on it. These people are called curators.
+Up to 25% of the reward for each publication goes to the people who voted on it. These people are called Curators.
```

Comment:
Steemit uses the word "post" everywhere, not "publication". Also, why is "curator" capitalized?

Reply:
digital currency => digital currency tokens 
IMHO redundant and confusing